update-rules.json

```diff
@@ -829,5 +828,286 @@
+     "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "#content > h3:nth-child(8)",
+        "baselink": "https://downloads.sourceforge.net/project/winscp/WinSCP/{{.version}}/WinSCP-{{.version}}-Setup.exe"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+    "processhacker": {
+     "url": "http://processhacker.sourceforge.net/downloads.php",
+     "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='setup.exe']"
+      }
+    },
+    "version_extractor": "([0-9].[0-9]{1,4})"
+  },
+   "everything-search": {
+    "url": "https://www.voidtools.com/downloads/",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "a[href$='x86.Multilingual-Setup.exe']"
+      },
+      "x86_64":{
+        "rule_type": "css-link",
+        "selector": "a[href$='x64.Multilingual-Setup.exe']"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "mpc-hc": {
+    "url": "https://mpc-hc.org/downloads/",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "a[href$='.x86.exe']"
+      },
+      "x86_64":{
+        "rule_type": "css-link",
+        "selector": "a[href$='.x64.exe']"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "deluge": {
+    "url": "http://dev.deluge-torrent.org/wiki/Download",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": "#wikipage > p:nth-child(2)",
+		"baselink": "http://download.deluge-torrent.org/windows/deluge-{{.version}}-win32-py2.7-0.exe"
+      }
+    },
+    "version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+  "sharex": {
+    "url": "https://github.com/ShareX/ShareX/releases/latest",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='setup.exe']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "lockhunter": {
+    "url": "http://lockhunter.com/download.htm",
+    "updater": {
+      "x86":{
+        "rule_type": "css-html-version",
+        "selector": ".download_comment",
+		"baselink": "http://lockhunter.com/exe/lockhuntersetup_{{.version-}}.exe"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "obs-studio": {
+    "url": "https://obsproject.com/download",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='Full-Installer.exe']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "colemak": {
+    "url": "https://colemak.com/Windows",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='Caps-Lock-Unchanged.zip']"
+      }
+    },
+	"version_extractor": "([.\\d]+[.]\\d{1,5})"
+  },
+   "handbrake": {
+    "url": "https://handbrake.fr/downloads.php",
+    "updater": {
+      "x86":{
+        "rule_type": "css-link",
+        "selector": "[href$='i686-Win_GUI.exe']"
```

Comment:
It looks like the naming convention changes from version to version. A suffix (e.g. "-1") is sometimes included in the file name, so a more generic selector would be better in order to get matches when a suffix is included.